### DISCUSSION TITLE
✅ Option to Copy text from the terminal

### DISCUSSION THREAD

**Discussion by p222pjp** [isontheline/pro.webssh.net#416] [Ideas]
Would be really useful if you could select the text in the Terminal window and copy to the clipboard.

**Comment by isontheline** [comment]
Hello @p222pjp 👋

The current workaround is to long press the terminal in order to popup a menu that let you show the text selection.

There is an issue for improving the terminal text selection : #144

**Comment by p222pjp** [comment]
Thank you so much for the quick response.

This workaround will be fine for now thank you.

Any improvement to be able to do it straight away would be great. Will keep not on issue 144.

**Reply by isontheline** [reply]
Thank you for your feedback @p222pjp 🙏
